.gitignore

```diff
@@ -44,8 +44,7 @@ object_storage/
 __emails__
 share/GeoLite2-City.mmdb
 a11y/
-# pyenv local config
-.python-version
```

Comment:
- This `.python-version` is already excluded on line 36 🤷.
- The comment adds little value as the only comment in the file.

Reply:
_Some_ context is always better than _no_ context. If anything, we should be making sure other entries have comments, rather than removing the lone bit of useful info. Can you move the comment to the remaining `.python-version` line?